modules/application/src/main/java/edu/northeastern/truthtree/adapter/utilities/URLUtil.java

```diff
@@ -54,4 +62,49 @@ private static String readURL(String url) {
 
     return response.toString();
   }
+
+  /**
+   * Reads a web page and turns its contents into a string.
+   *
+   * @param url The web address of the web page to be read and converted.
+   * @return Web contents as a string
+   */
+  public static String postJSONFromURL(String url, String jsonString) {
+    StringBuffer response = new StringBuffer();
+    try {
+      URL postUrl = new URL(url);
+      HttpURLConnection con = (HttpURLConnection) postUrl.openConnection();
+      con.setRequestMethod("POST");
+      con.setDoOutput(true);
+      con.setRequestProperty("Content-Type", "application/json");
+      DataOutputStream write = null;
+      try {
+        write = new DataOutputStream(con.getOutputStream());
+        write.writeBytes(jsonString);
+        BufferedReader in = new BufferedReader(
+                new InputStreamReader(con.getInputStream()));
+        String output;
+        response = new StringBuffer();
+        while ((output = in.readLine()) != null) {
+          response.append(output);
+        }
+        System.out.println(response);
+        in.close();
+      } catch (IOException exception) {
+        throw exception;
+      } finally {
+        try {
+          if (write != null) {
+            write.close();
+          }
+        } catch (IOException ex) {
```

Comment:
Exception handling at controller level?

Reply:
Exception Handling is not done yet for any api. Will have to add custom exception or at least throw Java exception till controller.

Reply:
You can just let it propagates and catch it at Stories controller and send the status code accordingly. Very simplistic example https://github.com/TruthTreeASD/backend/blob/develop/modules/application/src/main/java/edu/northeastern/truthtree/controller/TypeaheadController.java#L46

Reply:
I have done this before. I am saying that I'll add exception framework later on. Sheela needed an urgent push and that's why I am pushing what is working for now.